provision/acc_provision/acc_provision.py

```diff
@@ -380,6 +381,12 @@ def normalize_cidr(cidr):
     return str(n)
 
 
+def get_md5_hash(str):
```

Comment:
This PR is for Issue - https://github.com/noironetworks/support/issues/1952 (Referring issue in the comment for context purpose only)

Is the change required only for cko-mvp-1 branch (as per the issue) ?
Or its needed for master and kmr2 branches as well ?